tee-worker/identity/client-api/parachain-api/prepare-build/interfaces/trusted_operations/definitions.ts

```diff
@@ -25,7 +32,7 @@ export default {
         },
         TrustedCallAuthenticated: {
             call: "TrustedCall",
-            nonce: "Index",
+            nonce: "u32",
```

Comment:
it should work with `Index` too. I mean, without changing anything else – did you find any issues? it'd be good to keep the type consistent with the other `nonce` in these definitions.

Reply:
We can also change both to `Index`. What do you think?

Reply:
Historically it was called `index`, not it's `nonce`

Reply:
Right, I was confused by the enclave nonce's getter and `system.accountNextIndex` – both return `Index`.

> We can also change both to Index. What do you think?

Sounds good to me